src/lib/methods/delete-item.ts

```diff
@@ -1,13 +1,16 @@
 import { DeleteItemInput } from 'aws-sdk/clients/dynamodb';
+import retry, { Options } from 'p-retry';
```

Comment:
Rename `Options` to `RetryOptions` here.

Reply:
Will remove `p-retry` options as argument. Only accepting numbers now to make the API more user friendly.